operators/pkg/tenant-controller/tenant_controller.go

```diff
@@ -320,6 +404,7 @@ func (r *TenantReconciler) createOrUpdateClusterResources(ctx context.Context, t
 	}
 
 	var retErr error
+
```

Comment:
This change shall be reverted.

Reply:
Is "this change" specifically referring to the newline added?

Reply:
Yes

Reply:
Resolved in 
[f35a986](https://github.com/netgroup-polito/CrownLabs/pull/821/commits/f35a986a653da142c65add7ed60fd0deaf49ecfc)